src/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Restaurant Page!</title>
+</head>
+<body>
+    <nav class="amber">
+
+    </nav>
+      <main>
+        <div id="content">
+            
+        </div>
+      </main>

```

Comment:
Well done developer.... 👍 
According to the instruction on `Odin` page, we should only have `<div id="content"></div>` in side the `body` tag. so you need to remove the `nav` tag added.
![image](https://user-images.githubusercontent.com/31897434/88849163-66288600-d1e1-11ea-8414-32df71189e88.png)
https://www.theodinproject.com/courses/javascript/lessons/restaurant-page